src/UUID/DynamicUUID.php

```diff
@@ -1,17 +1,13 @@
 <?php
 
 
-namespace Paynl;
+namespace Paynl\UUID;
```

Comment:
Changing the namespace will cause problems with users already using this classes